core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -120,6 +105,15 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
   }
 
   override def receivePluginInternal: Receive = {
+    case tw: TagWrites =>
+      log.debug("Sending tag writes: {}", tw)
+      tagActor(tw.tag) ! tw
+    case tws: Vector[TagWrites] =>
```

Comment:
collections are not good actor messages, since the type is erased, add another case class.
but it could be worth optimizing for the the common case, which is only one tag and then just send  `TagWrites`

Reply:
Yeah this is dumb, fixed